src/components/test-component.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+function TestComponent(props) {
```

Comment:
Hmmm, weird, I forgot to put "export" here.  And yet, in the test code, I was able to import it.  I didn't think es6 modules would allow that.